libexec/bats-core/bats

```diff
@@ -279,6 +279,15 @@ done
 # shellcheck source=lib/bats-core/validator.bash
 source "$BATS_ROOT/lib/bats-core/validator.bash"
 
+# Prompt to show in verbose results: distinguish between root and non
+if [ -z "$BATS_PS1" ]; then
```

Comment:
What kind of problem are you trying to solve here? Shouldn't users be aware whether they ran their tests under root or not?

Reply:
Actually, that too is invaluable in our project CI: the [podman](https://github.com/containers/podman) tool is internally very different between root and rootless contexts, and often the most important first question when looking at a failure is: root, or rootless? (We expose that in other places in CI logs, but the prompt makes it even easier). I'm not strongly committed to this, though.